netlify.toml

```diff
@@ -8,4 +8,7 @@
   environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "staging", REACT_APP_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", REACT_APP_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_SANITY_DATASET = "staging", GATSBY_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_INTERCOM_APP_ID = "kn5di10s", REACT_APP_LEDGER_CHAIN_ID = "regen-hambach-1", REACT_APP_LEDGER_CHAIN_NAME = "Regen - Hambach Testnet", REACT_APP_BLOCK_EXPLORER = "https://hambach.regen.aneka.io", REACT_APP_EXP_LEDGER = "true", REACT_APP_NCT_BASKET = "true" }
 
 [context.redwood]
-  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "staging", REACT_APP_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", REACT_APP_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_SANITY_DATASET = "staging", GATSBY_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_INTERCOM_APP_ID = "kn5di10s", REACT_APP_LEDGER_CHAIN_ID = "regen-redwood-1", REACT_APP_LEDGER_CHAIN_NAME = "Regen - Redwood Testnet", REACT_APP_BLOCK_EXPLORER = "https://redwood.regen.aneka.io", REACT_APP_NCT_BASKET = "true" }
\ No newline at end of file
+  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "staging", REACT_APP_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", REACT_APP_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_SANITY_DATASET = "staging", GATSBY_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_INTERCOM_APP_ID = "kn5di10s", REACT_APP_LEDGER_CHAIN_ID = "regen-redwood-1", REACT_APP_LEDGER_CHAIN_NAME = "Regen - Redwood Testnet", REACT_APP_BLOCK_EXPLORER = "https://redwood.regen.aneka.io", REACT_APP_NCT_BASKET = "true" }
+
+[build.environment]
+  NODE_OPTIONS = "--max_old_space_size=4096"
```

Comment:
added this because we were having some memory allocation issues: https://app.netlify.com/sites/regen-registry/deploys/62a86b31a1cb0f000847762d 
That started to happen once I added @keplr-wallet/cosmos dependency which we use for validating bech32 addresses. Btw validating bech32 addresses on other forms (eg credits send) should be tackled separately if we want to do that.